electrum/gui/qt/staking_detail_tx_window.py

```diff
@@ -705,28 +720,28 @@ def insert_data(self, vbox):
         hbox_payout = QHBoxLayout()
         p_lab = QLabel(_("Payout") + ':')
         hbox_payout.addWidget(p_lab)
-        p_lab_data = QLabel(_("0.000000003 ELCASH"))
+        p_lab_data = QLabel(f"{self.data.staking_info.staking_amount*0.97} ELCASH")
         hbox_payout.addWidget(p_lab_data)
         vbox_right.addLayout(hbox_payout)
 
         hbox_blocks = QHBoxLayout()
         blocks_lab = QLabel('Block time:')
-        blocks_lab_data = QLabel('15:11')
+        blocks_lab_data = QLabel('??:??')
         hbox_blocks.addWidget(blocks_lab)
         hbox_blocks.addWidget(blocks_lab_data)
         vbox_right.addLayout(hbox_blocks)
 
         hbox_fee = QHBoxLayout()
         p_lab = QLabel(_("Number od tx:"))
         hbox_fee.addWidget(p_lab)
-        p_lab_data = QLabel(_("-3"))
+        p_lab_data = QLabel(_("1"))
         hbox_fee.addWidget(p_lab_data)
         vbox_right.addLayout(hbox_fee)
 
         hbox_fee = QHBoxLayout()
         p_lab = QLabel(_("Penalty:"))
         hbox_fee.addWidget(p_lab)
-        p_lab_data = QLabel(_("-3.001 ELCASH"))
+        p_lab_data = QLabel(self.data.staking_info.staking_amount*(-0.03))
```

Comment:
this percentage could be taken from `network.staking_info['penalty']`
and also we need to grep through the code and get rid of any hardcoded penalties and use network-provided penalty params